pytorch_lightning/accelerators/accelerator_connector.py

```diff
@@ -148,6 +147,7 @@ def __init__(
 
     def handle_given_plugins(self, plugins: Optional[Sequence]):
         if plugins is None:
+            self._cluster_environment = self.select_cluster_environment()
```

Comment:
Instead of this you could just initialize an empty list:

```python
plugins = plugins if plugins is not None else []
```

This way you don't need to have the return or this additional cluster environment line

Reply:
Yes.